apps/web/screens/OfficialJournalOfIceland/OJOISubpage.tsx

```
@@ -14,30 +17,36 @@ import { CustomNextError } from '@island.is/web/units/errors'
 import { webRichText } from '@island.is/web/utils/richText'
 
 import { OJOIWrapper } from '../../components/OfficialJournalOfIceland'
-import { CustomScreen, withCustomSubpageWrapper } from '../CustomPage'
+import { CustomScreen, withCustomPageWrapper } from '../CustomPage'
 import { GET_ORGANIZATION_QUERY } from '../queries'
-import { OJOI_BASE_CMS_ID, ORGANIZATION_SLUG } from './constants'
+import { GET_CUSTOM_SUBPAGE_QUERY } from '../queries/CustomPage'
+import { ORGANIZATION_SLUG } from './constants'
 import { m } from './messages'
 
-interface OJOIHelpProps {
+interface OJOISubpageProps {
   organization?: Query['getOrganization']
   locale: Locale
+  customSubpage?: CustomPage
 }
 
-const OJOIHelp: CustomScreen<OJOIHelpProps> = ({
+const OJOISubpage: CustomScreen<OJOISubpageProps> = ({
   organization,
-  customPageData,
   locale,
+  customSubpage,
 }) => {
   const { formatMessage } = useIntl()
   const { linkResolver } = useLinkResolver()
 
+  if (!customSubpage) {
+    throw new CustomNextError(404, 'Custom subpage not found')
```

Wouldn't it be better to throw this error in the getProps function below?